engine/src/main/java/org/destinationsol/game/FactionInfo.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2018 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.destinationsol.game;
+
+import com.google.gson.JsonArray;
+import com.google.gson.JsonParser;
+import org.destinationsol.game.input.Pilot;
+import org.destinationsol.game.ship.SolShip;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FilenameFilter;
+import java.util.ArrayList;
+
+public class FactionInfo {
+
+    private static ArrayList<String> factionName = new ArrayList<String>();
+    private static ArrayList<String> factionColor = new ArrayList<String>();
+    private static ArrayList<Integer> factionDisposition = new ArrayList<Integer>();
+    public FactionInfo() {
+        createFactionList();
+
+    }
+
+    private void createFactionList(){
+        String[] folderList = getFolderList();
+
+        for(int i = 0; i < folderList.length; i++) {
+            File factionFile = new File("modules/" + folderList[i] + "/assets/configs/factions.JSON");
```

Comment:
Please use `Assets` instead of directly retrieving the file ... and generally rework this method to use `Assets`